client/client.go

```diff
@@ -155,6 +155,9 @@ type Config struct {
 
 	// UserAgent is the User-Agent header sent to the Pebble daemon.
 	UserAgent string
+
+	// ContentType is the Content-Type header sent to the Pebble daemon.
+	ContentType string
```

Comment:
I don't think we need to make this configurable: for Pebble API requests, it should always be `application/json`.